components/clarinet-format/tests/golden-intended/clarity-bitcoin.clar

```diff
@@ -0,0 +1,1183 @@
+;; source: https://github.com/hirosystems/clarity-examples/blob/main/examples/clarity-bitcoin/contracts/clarity-bitcoin.clar
+;; @contract stateless contract to verify bitcoin transaction
+;; @version 5
+;; version 5 adds support for txid generation and improves security
+;; Error codes
+(define-constant ERR-OUT-OF-BOUNDS u1)
+(define-constant ERR-TOO-MANY-TXINS u2)
+(define-constant ERR-TOO-MANY-TXOUTS u3)
+(define-constant ERR-VARSLICE-TOO-LONG u4)
+(define-constant ERR-BAD-HEADER u5)
+(define-constant ERR-HEADER-HEIGHT-MISMATCH u6)
+(define-constant ERR-INVALID-MERKLE-PROOF u7)
+(define-constant ERR-PROOF-TOO-SHORT u8)
+(define-constant ERR-TOO-MANY-WITNESSES u9)
+(define-constant ERR-INVALID-COMMITMENT u10)
+(define-constant ERR-WITNESS-TX-NOT-IN-COMMITMENT u11)
+(define-constant ERR-NOT-SEGWIT-TRANSACTION u12)
+(define-constant ERR-LEFTOVER-DATA u13)
+;;
+;; Helper functions to parse bitcoin transactions
+;;
+;; Create a list with n elments `true`. n must be smaller than 9.
+(define-private (bool-list-of-len
+		(n uint)
+	)
+	(unwrap-panic (slice? (list true true true true true true true true) u0 n))
+)
+
+;; Reads the next two bytes from txbuff as a little-endian 16-bit integer, and updates the index.
+;; Returns (ok { uint16: uint, ctx: { txbuff: (buff 4096), index: uint } }) on success.
+;; Returns (err ERR-OUT-OF-BOUNDS) if we read past the end of txbuff
+(define-read-only (read-uint8
+		(ctx {
+				txbuff: (buff 4096),
+				index: uint
+			}
+		)
+	)
+	(let (
+		(data (get txbuff ctx))
+		(base (get index ctx))
+	)
+		(ok
+			{
```

Comment:
Could/should the new line before `{` be avoided?
Looks like there's this pattern in all `(ok <tuple>)`

Reply:
it depends on the source. if it's over max width it'll newline it. This could be changed but it starts looking weird if you start a map on the same line as `(ok ..` and then dump all the rest of the keys on the next indented lines

```
(ok { something: 1,
  a: b,
  c: d
  }
)
```

Oh, maybe I see your point. It should be this?

```
(ok {
  something: 1,
 ```

Reply:
Yes that was my point
```
(ok {
  k1: 1
  k2: 2
})
```

Instead of
(ok
  {
    k1: 1
    k2: 2
  }
)
```